src/pages/stop/stop.component.ts

```diff
@@ -163,7 +178,12 @@ export class StopComponent {
   toggleStopHeart(): void {
     // console.log('toggling', stop.Description);
     this.favoriteStopSvc.toggleFavorite(this.stopId, this.stop.Description);
+    if (this.liked) {
+      this.showToast('Stop added to Favorites');
+    }
+    else this.showToast('Stop removed from Favorites');
```

Comment:
So I have the same questions about the various repetitions of this logic, both setting the `toast` property of the component, and defining the message logic. Is there a way of abstracting it upwards?